script.js

```diff
@@ -297,18 +325,38 @@ const usersList = {
           click: () => {
             $$("usersList").sort("name", "desc");
           }
+        },
+        {
+          view: "button",
+          label: "Add new",
+          gravity: 0.2,
+          css: "webix_primary",
+          click: () => {
+            const name = $$("listFilter").getValue();
+            const usersList = $$("usersList");
+            $$("usersList").add({
```

Comment:
if there're a lot of items in the list, the new one is added in the last position and is not visible, which is inconvenient. try to: 

- either add an item into the 1st position
- or scroll to it by API

Reply:
Ready to review.

Reply:
we use adding  also  in the first tab (form+datatable)